tests/data/prometheus_metrics.json

```diff
@@ -2,13 +2,6 @@
   "status": "success",
   "data": [
     "cni_wait_connectivity_seconds_total",
-    "felix_exec_time_micros_count",
-    "felix_exec_time_micros_sum",
-    "felix_ipset_calls",
-    "felix_ipset_errors",
-    "felix_ipset_lines_executed",
-    "felix_ipsets_calico",
-    "felix_ipsets_total",
```

Comment:
@mateoflorido after several runs of this upgrade, these metrics no longer appear to be populated by kube-ovn.  I can't tell if kube-ovn has dropped these calico style metrics or really where this entire `prometheus_metrics.json` comes from.

Reply:
i did find this [list of metrics](https://raw.githubusercontent.com/kubeovn/docs/v1.12/docs/reference/metrics.en.md) from the kubeovn docs repo though